### PR TITLE
Fix template example 8

### PR DESCRIPTION
**Pull request by esambo**
## Description
Some necessary and optional changes to get the template example to work.
Please see the git commit messages for more context.
I kept the commits very small and ordered them in priority, where the first several ones were essential to get the examples to even work, and the later ones were just helpful and could be optional. Feel free to revert or cherry-pick individual commits if necessary (eg. the last one).

## Screenshot
The header now also shows a new menu bar entry of "Change Auth" when logged in, which navigates to the shown page:
Screen Shot 2020-08-25 at 4 09 27 PM

### CODE REVIEW

**Comment by LarryKlugerDS** [comment]
Thank you for the PR!! I have submitted DEVDOCS-3214 to have our engineering team review your PR.

Thanks again, Larry

File: app/views/layouts/_head.erb [hunk 1]
```diff
@@ -40,6 +40,10 @@
         <a class="nav-link" href="/ds/logout" id="logout"
            data-busy="href">Logout <span class="sr-only">(current)</span></a>
```
[the remark this answers is not in the document]
Comment (esambo) [reply app/views/layouts/_head.erb]: @AaronWDS I didn't change anything with the "Logout" link and only added the "Change Auth" link in 0985d3f two lines further below. I reverted it in bde6f0a

File: README.md [hunk 2]
```diff
@@ -64,7 +65,7 @@ This launcher includes code examples for the following workflows:
    metadata with the envelope.
    [Source.](./app/services/eg016_service.rb)
 1. **Set template tab values.**
-   The example creates an envelope using a template and sets the initial values for its tabs (fields).
```
[the remark this answers is not in the document]
Comment (esambo) [reply README.md]: Indeed, I am so sorry about that! I guess I was just overwhelmed by all the newly created tabs. I reverted it in 9de37c8

File: README.md [hunk 3]
```diff
@@ -183,13 +183,15 @@ When the token expires, it updates automatically.
...
-2. Create a new API key in the Admin panel: https://admindemo.docusign.com/api-integrator-key, take note of the public key.
-3. Set a redirect URI of `http://localhost:3000/auth/docusign/callback` as mentioned in the installation steps above for the API key you make in step 2.
-4. Generate an RSA keypair in the administrator console on the DocuSign developer sandbox and copy the private key to a secure location.
-5. Create a new file in the config folder named **docusign_private_key.txt**, and paste in that copied RSA private key, then save it.
-6. Update the file **config/appsettings.yml** and include the newly created API key from step 2 as well as your account user id GUID which is also found on the Admin panel: `https://admindemo.docusign.com/api-integrator-key`.
-
-From there you should be able to run the launcher using **rails s** then selecting **JSON Web Token** when authenticaing your account.
+2. Create a new API key in the Admin panel: https://admindemo.docusign.com/api-integrator-key
+  - Take note of the _Integration Key_.
+  - Create a _Secret Key_ and take note of that too.
```
[the remark this answers is not in the document]
Comment (esambo) [reply README.md]: Yes, that makes sense. I removed it in 4203d0c

**Review by esambo — COMMENTED** [review]
Thanks @AaronWDS for the feedback! I believe that I addressed all of them, although I was a bit confused about the one related to the header, as you commented on an existing line

**Review by AaronWDS — APPROVED** [review]
lgtm

**Pull request merged**